src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -110,15 +112,28 @@ export class MeetingManager implements AudioVideoObserver {
 
   simulcastEnabled: boolean = false;
 
+  videoDownlinkBandwidthPolicy: VideoDownlinkBandwidthPolicy | null = null;
```

Comment:
@ltrung should we start setting this to `undefined` instead of `| null`. In the recent JS SDK version we are pushing to use `undefined` instead of `null`. Want to know your opinion. Sorry for the late comment @xuesichao .

https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#null-and-undefined

Reply:
The guidelines apply for the TypeScript contributors but checking on what we should use.

Reply:
Yes, we should prefer `undefined` over `null`. Adding `| null` can make a variable to have three types of values unnecessarily: e.g., `VideoDownlinkBandwidthPolicy`, `undefined`, and `null`.